specification/botservice/resource-manager/Microsoft.BotService/stable/2020-06-02/examples/PutConnection.json

```diff
@@ -0,0 +1,81 @@
+{
+  "parameters": {
+    "subscriptionId": "subscription-id",
+    "resourceGroupName": "OneResourceGroupName",
+    "api-version": "2020-06-02",
+    "resourceName": "samplebotname",
+    "connectionName": "sampleConnection",
+    "parameters": {
+      "location": "West US",
+      "etag": "etag1",
+      "properties": {
+        "clientId": "sampleclientid",
+        "clientSecret": "samplesecret",
+        "scopes": "samplescope",
+        "serviceProviderId": "serviceproviderid",
+        "parameters": [
+          {
+            "key": "key1",
+            "value": "value1"
+          },
+          {
+            "key": "key2",
+            "value": "value2"
+          }
+        ]
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "location": "global",
+        "id": "/subscriptions/subscription-id/resourceGroups/OneResourceGroupName/providers/Microsoft.BotService/botServices/samplebotname/connections/sampleConnection",
+        "etag": "etag1",
+        "name": "samplebotname/sampleConnection",
```

Comment:
The name in the response for nested resource should be the last segment of the id. In this case, it should be `sampleConnection` instead of `samplebotname/sampleConnection`.

(In ARM templates are the `name` is used differently and `samplebotname/sampleConnection` would be ok there when deploying a nested resource.)